lib/docsparser.js

```diff
@@ -180,7 +193,9 @@ DocsParser.prototype.format_function = function(name, args, retval, options) {
         format_args.forEach(function(element) {
             format_str = util.format(format_str, element);
         });
-        out.push(format_str);
+        if(name !== 'constructor') {
```

Comment:
Instead of not add the `format_str` if the function is a constructor, could you please change ` if(ret_type !== null) {` to ` if(ret_type !== null && name !== 'constructor') {`. This way the hole return part can be skipped.